tasks/Dockerfile

```diff
@@ -40,6 +40,8 @@ RUN dnf -y update && \
         rpmdevtools \
         rsync \
         sassc \
+        selinux-policy \
+        selinux-policy-devel \
```

Comment:
Dou you mind moving that after the `builddep` with a HACK comment pointing to the cockpit PR? We can drop this once it lands in fedora, as then builddep will install it  for us.

Reply:
Right, good call. Will do.